### ISSUE TITLE
cython automatically downgrades to python2, even if compiled in python3 env

### ISSUE THREAD

**Issue by scalavision** [pysam-developers/pysam#893]
I get this in my log when installing pysam in `nixpkgs` using python3:

```s
Cython directive 'language_level' not set, using 2 for now (Py2). This will change in a later release!
```

This SO explains how to avoid it, but I was not able to make it work. 

* <https://stackoverflow.com/questions/34603628/how-to-specify-python-3-source-in-cythons-setup-py>

Maybe you can look at it and see if it is possible to do something similar?

**Comment by jmarshall** [comment]
Yes, I too had a go at making this warning disappear via `language_level` last year, but without much success.

What are pysam's plans around stopping supporting Python 2.x? It seems like using this directive will become simpler once there is only Python 3…

**Comment by jmarshall** [comment]
Pysam now supports only Python 3.x and uses `language_level=3` throughout. The last few of these caused the warning during `pytest` and were fixed in f3260442d7b5477492e371288716963bcb12fad5.
